### PR TITLE
3D touch & FLEX

### PR DESCRIPTION
**Pull request by bolencki13**
3D Touch support for packages and their respective detail pages. FLEX support to find details about classes and view HTML in UIWebViews. 3D Code by @KJCYOLORhymes_twitter.

### CODE REVIEW

**Comment by JohnCoates** [comment]
The 3D Touch code I'm all for, but instead of committing the source code of FLEX how about we add it in as a CocoaPod?

**Comment by bolencki13** [comment]
Convert to a CocoaPod project (*.xcworkspace) or just drop in the library?

**Comment by JohnCoates** [comment]
Let's not include the Pods/ directory into the repo, as that brings in all the source code for FLEX and a major benefit of having the Podfile.lock file is to not have to include that.  

With this pull request FLEX is being included in Release builds. We definitely don't want to distribute Limitless with FLEX integrated into it.  

Personally, I use [FLEXing](https://github.com/ThePantsThief/FLEXing) so that FLEX is injected into all apps. So I'd rather not compile the extra 76 FLEX source code files every time I clean the project. I'm thinking of two options then: Let's either add a new Configuration named Development, or since people need to run `pod install` to use it anyways, let's just add the Podfile and the Podfile.lock and leave FLEX to be integrated into the project when that command is run.

**Comment by bolencki13** [comment]
Completely removed FLEX support

File: Limitless.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2958,7 +2958,7 @@
 				CLANG_ENABLE_MODULES = NO;
 				CLANG_ENABLE_OBJC_ARC = NO;
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
-				DEVELOPMENT_TEAM = E6QK4MU6LM;
+				DEVELOPMENT_TEAM = 9UKYMVL752;
```
Comment (JohnCoates) [inline Limitless.xcodeproj/project.pbxproj:2961]: In the future don't commit changes to DEVELOPMENT_TEAM or we'll be ping-ponging this endlessly.

**Comment by JohnCoates** [comment]
Alright, looks good. I'm all for FLEX support. I think it would be wise to write some instructions up in Markdown for the Documentation directory so we can link to it in the Readme in case anyone wants to use FLEX.

**Pull request merged**